OPCB/functions/missions/GDrunken.sqf

```diff
@@ -50,20 +45,22 @@ publicVariable "OPCB_econ_credits";
 
 [_base] call CHAB_fnc_endmission;
 [ _comp ] call LARs_fnc_deleteComp;
+
 {
   [ _x ] call LARs_fnc_deleteComp;
 } forEach (_stations select 0);
 
-{
-	if(typeName _x == "GROUP") then
+[_reference, _trucks] spawn {
+	params ["_reference", "_trucks"];
 	{
-		{
-		  deleteVehicle _x;
-		} forEach (units _x);
-		deleteGroup _x;
-	}
-	else{
-		deleteVehicle _x;
-	};
-} forEach _reference;
-
+		if(typeName _x == "GROUP") then {
+			{
+				deleteVehicle _x;
+			} forEach (units _x);
+			deleteGroup _x;
+		}
+		else{
+			deleteVehicle _x;
+		};
+	} forEach _reference append _trucks;
```

Comment:
`append` doesn't return anything. This won't work.

Reply:
It might not work, true. 
But it also doesn't generate an error since I have managed to spawn the task and also finish it AND make the vehicles disappear